op-verifier/deploy_l1/00_op_verifier.ts

```diff
@@ -3,16 +3,22 @@ import {DeployFunction} from 'hardhat-deploy/types';
 import fs from 'fs';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const {deployments, getNamedAccounts} = hre;
+  const {deployments, getNamedAccounts, network} = hre;
   const {deploy} = deployments;
-
   const {deployer} = await getNamedAccounts();
-
-  const opAddresses = await (await fetch("http://localhost:8080/addresses.json")).json();
-
+  let L2_OUTPUT_ORACLE_ADDRESS, GATEWAY_URL
+  if(network.name === 'opDevnetL1'){
```

Comment:
Can you use a mapping of URLs instead of an if statement here?

Reply:
I still need if statement to fetch L2_OUTPUT_ORACLE_ADDRESS address dynamically though